src/main/java/io/cdap/e2e/utils/StorageClient.java

```diff
@@ -87,10 +95,18 @@ public static Bucket createBucket(String bucketName) throws IOException {
   }
 
   public static Blob uploadObject(String bucketName, String objectName, String filePath)
-    throws IOException, URISyntaxException {
+      throws IOException, URISyntaxException {
     return getStorageService().create(
-      BlobInfo.newBuilder(BlobId.of(bucketName, objectName)).build(),
-      Files.readAllBytes(Paths.get(StorageClient.class.getResource("/" + filePath).toURI())));
+        BlobInfo.newBuilder(BlobId.of(bucketName, objectName)).build(),
+        Files.readAllBytes(Paths.get(StorageClient.class.getResource("/" + filePath).toURI())));
   }
 
+  public static Bucket createBucketwithLifeCycle(String bucketName, int age) throws IOException {
+    return getStorageService().create(BucketInfo.of(bucketName)).toBuilder()
+        .setLifecycleRules(
+            ImmutableList.of(
+                new LifecycleRule(
+                    LifecycleAction.newAbortIncompleteMPUploadAction(),
+                    LifecycleCondition.newBuilder().setAge(age).build()))).build().update(); }
```

Comment:
Add the curly braces below and there should be a space between the final and  this curly braces after you change .
     LifecycleCondition.newBuilder().setAge(age).build()))).build().update();
    }

}

Reply:
resolve this .

Reply:
Changes has been done , as per the review comment